config/database.yml

```diff
@@ -58,6 +60,8 @@ development:
 test:
   <<: *default
   database: rails_recipe_app_test
+  username: postgres
+  password: pw4Admin
```

Comment:
It can be better if we keep the databases config file without any username and password because those are going to be different from one computer to another.